components/mixins/experience-event/experienceevent-card-application-process.schema.json

```diff
@@ -20,20 +22,15 @@
         "xdm:personalFinances": {
```

Comment:
There is still no nesting here. This "personalFinances" is just a definition not an object.